addons/google_calendar/models/google_sync.py

```diff
@@ -179,6 +179,9 @@ def _sync_google2odoo(self, google_events: GoogleEvent, default_reminders=()):
             # Migration from 13.4 does not fill write_date. Therefore, we force the update from Google.
             if not odoo_record.write_date or updated >= pytz.utc.localize(odoo_record.write_date):
                 vals = dict(self._odoo_values(gevent, default_reminders), need_sync=False)
+                # don't override existing videocall_location's value with False
+                if 'videocall_location' in odoo_record and 'videocall_location' in vals and not vals['videocall_location']:
+                    vals.pop('videocall_location')
```

Comment:
Hello :wave: Thanks for your PR. I understand the issue but in my opinion, the change should be made in the `_odoo_values` method in `calendar.py`. This method translate the google values into Odoo one. 
Would you agree to modify the code there ? Thanks :-).

Reply:
My first thought was there in `_odoo_value` too but later I rethought that there could be a case that needed information from google. If we poped that video location there, that could be not so convenient for the others when they need it.

How do you think?

Reply:
ok, I have revised according to your comments. Tks!